lib/Screens/Questionarie/Charts/chart_promis_screen.dart

```diff
@@ -30,12 +34,22 @@ class _ChartPromisScreenState extends State<ChartPromisScreen> {
     {'dominionName': 'Funcionamento da personalidade', 'dominionNumber': 12},
     {'dominionName': 'Uso de Substância', 'dominionNumber': 13},
   ];
+  String legend = "";
 
   @override
   void initState() {
+    getLegend(QuestionnaireCode.pn1.name);
```

Comment:
Ta certo mesmo? nao seria o codigo?

Reply:
Tá certo. Se não quiser assim no initstate eu posso fazer igual no chart general e colocar um future builder.

Reply:
Digo o conteudo do atributo name é o que? o codigo do questionario? se for, troca o nome do atributo pra code

Reply:
Questionnaire.pn1.name resulta em uma string pn1. O name é uma função pra pegar o nome em string